src/main/java/seedu/address/model/company/exceptions/CompanyAlreadyMarkedOrUnmarked.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.company.exceptions;
+
+/**
+ * Signals that the operation will result in marking or unmarking a company that is already marked or unmarked.
+ */
+public class CompanyAlreadyMarkedOrUnmarked extends RuntimeException {
+    public CompanyAlreadyMarkedOrUnmarked() {
+        super("Operation would result in marking or unmarking a company that is already marked or unmarked");
+    }
```

Comment:
Maybe we can rename the file to CompanyAlreadyMarkedOrUnmarkedException?
Alternatively we can consider splitting into two separate exceptions for marked and unmarked so that the messages are clearer?